setup.py

```diff
@@ -19,7 +19,7 @@
 
 BUILD_FLAVOR = os.environ["BUILD_FLAVOR"] if "BUILD_FLAVOR" in os.environ else ""
 BUILD_NUMBER = os.environ["BUILD_NUMBER"] if "BUILD_NUMBER" in os.environ else ""
-VERSION = "1.8.0.4" + BUILD_FLAVOR + BUILD_NUMBER
+VERSION = "1.8.1" + BUILD_FLAVOR + BUILD_NUMBER
```

Comment:
I'd better not change naming scheme now, not sure how existing deps like `kensu-py>=1.8.0.4` would behave. so suggest something like `1.8.1.0`